v3/pforzheim.py

```diff
@@ -0,0 +1,113 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+import csv
+import json
+from datetime import datetime, timezone
+from decimal import Decimal
+from io import StringIO
+from typing import Optional
+
+from validataclass.dataclasses import validataclass
+from validataclass.exceptions import ValidationError
+from validataclass.validators import BooleanValidator, DataclassValidator, DecimalValidator, IntegerValidator, StringValidator
+
+from common.base_converter import JsonConverter
+from common.exceptions import ImportParkingSiteException
+from common.models import ImportSourceResult
+from common.validators import StaticParkingSiteInput
+from common.validators.base_validators import ParkingSiteTypeInput
+from common.validators.fields.noneable import ExcelNoneable
+from util import SourceInfo, name_to_id
+
+
+@validataclass
+class PforzheimRowInput:
+    uid: str = StringValidator(max_length=255)
+    name: str = StringValidator(max_length=255)
+    operator_name: str = StringValidator(max_length=255)
+    address: str = StringValidator(max_length=255, multiline=True)
+    description: str = StringValidator(max_length=512, multiline=True)
+    type: str = StringValidator(max_length=255)
+    lat: Decimal = DecimalValidator(min_value=40, max_value=60)
```

Comment:
You can use a `NumericValidator` instead of casting it manually to strings and use a `DecimalValidator` here.